crates/cubecl-matmul/src/components/global/multi_stage/ordered/matmul.rs

```diff
@@ -185,8 +187,9 @@ where
         batch_offset: u32,
         #[comptime] config: Self::Config,
     ) -> Self::RhsLoader {
+        let layout = SimpleGlobalLayout::new(&rhs, config.global_memory_config(MatmulIdent::Rhs));
         SyncPartialLoader::<MP::Rhs, Self::Config, RL>::new(
-            rhs,
+            rhs.view(layout.into_virtual()),
```

Comment:
It would be cool to only have `rhs.view(layout)` maybe by receiving the layout as a generic param and calling `into_virtual` directly into the `view` function. Just syntax sugar.

Reply:
the reason I didn't do that is because the macro causes dodgy issues around generics, so you always need to manually specify it. So it was actually cleaner this way. This is also why I went with the method on `VirtualTensor`, to abstract away the manually specified type on `TensorView::new`.